PelotonEppSdkTests/EventCreateTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using PelotonEppSdk.Classes;
+using PelotonEppSdk.Models;
+
+namespace PelotonEppSdkTests
+{
+    [TestClass]
+    public class EventCreateTests : TestBase
+    {
+        private string duplicateFriendlyUrlPath = "NEWEVENT";
+
+        private static EventRequest GetBasicEventRequest()
+        {
+            var factory = new RequestFactory(32, "Peloton123", "PelonEppSdkTests", baseUri);
+            var request = factory.GetEventCreateRequest();
+            return request;
+        }
+
+        [TestMethod]
+        public void TestSuccessCreateEvent()
+        {
+            var createRequest = GetBasicEventRequest();
+            createRequest.Name = "Test";
+            createRequest.Description = "Test";
+            createRequest.FriendlyUrlPath = duplicateFriendlyUrlPath;
+            createRequest.StartDatetime = DateTime.UtcNow;
+            createRequest.EndDatetime = DateTime.UtcNow;
+            createRequest.TermsAndConditionsContent = "";
+            createRequest.RefundPolicyContent = "";
+
+            var errors = new Collection<string>();
+            if (!createRequest.TryValidate(errors))
+            {
+                foreach (var error in errors)
+                {
+                    Debug.WriteLine(error);
+                }
+            }
+
+            var result = createRequest.PostAsync().Result;
+            Debug.WriteLine(result.Message);
+            Debug.WriteLineIf((result.Errors != null && result.Errors.Count >= 1), string.Join("; ", result.Errors ?? new List<string>()));
+            Assert.IsTrue(result.Success);
+            Assert.AreEqual(0, result.MessageCode);
+        }
+
+        [TestMethod]
+        public void TestFailureCreateDuplicateEvent()
```

Comment:
This test is not designed to be a unit. It depends upon another unit test, which is not an ideal situation.
This test should be rebuilt so that it creates two events -- first, a unique one and then one that duplicates the first.